dispatch/apps/api/views.py

```diff
@@ -403,6 +404,10 @@ def get_queryset(self):
                 Q(description__icontains=q) |
                 Q(host__icontains=q)
             )
+        if pending:
+            queryset = queryset.filter(
+                Q(is_submission__exact=1)
```

Comment:
I completed this in my PR because we were both modifying the same thing. Feel free to modify as needed!